core/src/main/scala/io/chrisdavenport/monoids/Dual.scala

```diff
@@ -3,7 +3,7 @@ package io.chrisdavenport.monoids
 import cats._
 import cats.implicits._
 
-final case class Dual[A](getDual: A) extends AnyVal
+final case class Dual[A](getDual: A)
```

Comment:
Dotty seems to have troubles with generic classes extending `AnyVal` but once removed everything works

Reply:
Should seriously consider writing these as opaque types in dotty, and leave this in scala 2. The performance hit if we box here can be pretty brutal.